relayer/middleware/wallet/walletToolBox.ts

```diff
@@ -19,7 +19,13 @@ export function createWalletToolbox(
   }
   switch (chainId) {
     case wh.CHAIN_ID_SOLANA:
-      return createSolanaWalletToolBox(providers, bs58.decode(privateKey));
+      let secretKey;
+      try {
```

Comment:
nit: factor this out into a solana pkey parser helper or do the parsing earlier so it doesn't have to be redone